packages/vite/src/node/config.ts

```diff
@@ -346,6 +386,7 @@ export type ResolvedConfig = Readonly<
     worker: ResolveWorkerOptions
     appType: AppType
     experimental: ExperimentalOptions
+    resolvedServerPersistentCacheOptions: ResolvedServerPersistentCacheOptions | null
```

Comment:
I would remove the `resolved` prefix for consistency. 

Also I think it should go inside the experimental options

Reply:
Well we don't have a `ResolvedExperimentalOptions` so I hesitated to change the whole thing